components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/overview-trend/overview-trend.component.ts

```diff
@@ -212,7 +219,7 @@ export class OverviewTrendComponent implements OnChanges, OnDestroy  {
       .attr('class', 'dot-group-bg');
 
     enter.on('click', (line) => {
-      this.dateSelected.next(moment(line.report_time));
+      this.dateSelected.next(moment(line.report_time).format('YYYY-MM-DD'));
```

Comment:
Return only the year, month, and day. We don't want to return time zone or hour.